configs/kontron_sa67_a53_defconfig

```diff
@@ -40,6 +40,7 @@ CONFIG_SPL_SYS_MALLOC_SIMPLE=y
 CONFIG_SPL_SYS_MMCSD_RAW_MODE=y
 CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x1000
 CONFIG_SPL_DMA=y
+CONFIG_SPL_ENV_SUPPORT=y
```

Comment:
These config options are all part of the am62x_a53_usbdfu.config. Could you try to just drop any changes in this file and compile with that fragment?